samples/CosmosDBIntegration/CosmosDbIntegrationTutorial.md

```diff
@@ -0,0 +1,117 @@
+# Tutorial: Setup Redis Triggers for Azure Function in Visual Studio
+Article 08/08/2023 Nadia Bugarin, Riley MacDonald, Phoebe Owusu
```

Comment:
<!-- Typically contributors are added through the metadata and show up as contributors when published. -->